inflect.py

```diff
@@ -981,6 +981,10 @@ def make_pl_si_lists(lst, plending, siendingsize, dojoinstem=True):
 
 
 pl_sb_uninflected_endings = [
+    # UNCOUNTABLE NOUNS
+    "butter",
+    "cash",
+    "information",
```

Comment:
```suggestion
    "information",
    "lore",
```

Reply:
As you might guess, I just ran into this problem with the word `"lore"` turning into `"lores"`.

Reply:
@skylerberg It looks like the word `lore` is an unusual case.  It can be either countable or uncountable depending on the context, as with a few other words.  I've placed a couple of examples into the irregular word lists.

What do you think?  (and equally, would you be able to handle your use case with this suggested logic in place?)

Reply:
@jayaddison this will work for me. Thanks for looking into this and figuring out that is sometimes countable. I didn't realize that.

Reply:
No problem, thanks for confirming.